npm_and_yarn/helpers/build

```diff
@@ -20,4 +20,4 @@ cp -r \
   "$install_dir"
 
 cd "$install_dir"
-npm ci
+npm ci --no-audit --fetch-timeout=600000
```

Comment:
Don't know if this will actually fix things, docs say the timeout is already 5 mins which I'm not sure is true from the build output. https://docs.npmjs.com/cli/v7/using-npm/config#fetch-timeout

Reply:
https://docs.npmjs.com/cli/v7/using-npm/config#fetch-retries seems interesting too - we may want to raise that from the default of `2`.

Reply:
Yeah will try `5` and see if the build failure goes away 👍